io.c

```diff
@@ -3203,67 +3203,87 @@ io_getpartial(int argc, VALUE *argv, VALUE io, int no_exception, int nonblock)
 
 /*
  *  call-seq:
- *     ios.readpartial(maxlen)              -> string
- *     ios.readpartial(maxlen, outbuf)      -> outbuf
+ *    readpartial(maxlen)             -> new_string
```

Comment:
Maybe just `string` instead of `new_string` (since receiver isn't a string). Similar for `read`.

Reply:
Done.